src/style.css

```diff
@@ -96,15 +96,20 @@ div#addList {
   padding: 15px 10px;
 }
 
-div#clear {
+button#clear {
   display: flex;
+  width: 100%;
+  height: 50px;
   justify-content: center;
-  border: 1px solid lightgray;
-  color: grey;
   align-items: center;
-  padding: 20px;
-  font-size: medium;
-  background-color: #f6f6f6;
+  color: black;
+  font-weight: 600;
+}
+
+button#clear :hover {
+  /* stylelint-disable-next-line declaration-colon-space-after */
```

Comment:
- [ ] Kindly do not disable linters, fixing linters help us to write a clean, clear, and strong codebase 💪🏾

Reply:
- [ ] According to the previous reviewer, please avoid disabling linter rules as they are very useful to keep a consistent code format and ensure you're using best practices! 🤓